Dockerfile

```diff
@@ -38,17 +38,10 @@ ENV PATH="/usr/local/ollama/bin:${PATH}"
 
 
 # Pull a language model (see LICENSE_STABLELM2.txt)
-ARG MODEL=stablelm2:1.6b-zephyr
+ARG MODEL=openchat
```

Comment:
Here is the [Ollama documentation](https://ollama.ai/library/openchat) for this model.

> A family of open-source models trained on a wide variety of data, surpassing ChatGPT on various benchmarks. Updated to version 3.5-0106.

Reply:
I switched to openchat because the generated responses were way better and it reacted way better on the context. Also, even stablelm2:1.6b timeouted on my machine, so I did not have a working setup on my computer anyway. Still wonder what the best way to develop and test such a thing without always deploying to a cloud machine.

Reply:
There is also qwen https://ollama.com/library/qwen
But it my first test, the smalles 0.5b parameter made lots of grammar mistakes in german.

Reply:
For now I had no interesting results with qwen. Should probably come up with #29 again. Openchat can be forced in production since #35. So this change is not needed anymore.